qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg

```diff
@@ -230,3 +230,6 @@
                     cmd_useradd = 'useradd fsgqa && useradd 123456-fsgqa && useradd fsgqa2'
                     cmd_get_tmpfs = 'df -h | grep /dev/shm | gawk '{ print $2 }''
                     cmd_set_tmpfs = 'mount -o remount,size=%s /dev/shm'
+        - start_daemon_with_other_user:
```

Comment:
Think it more, as it's group permission specific, seems it's better to move to the original cfg.